### PR TITLE
Add apiKey field to config.json

### PR DESCRIPTION
**Pull request by jeinygroove**

### CODE REVIEW

File: src/main/java/com/google/alpollo/ConfigHelper.java [hunk 1]
```diff
@@ -11,32 +11,40 @@
...
    */
   private static final class ConfigInfo {
     private String projectID;
+    private String apiKey;
 
     public String getProjectID() {
       return projectID;
     }
+    public String getApiKey() { return apiKey; }
   }
```
Comment (RazvanBudacaTechquarter) [inline src/main/java/com/google/alpollo/ConfigHelper.java]: I don't think it's a good idea to have different formatting on your getters.

File: src/main/java/com/google/alpollo/ConfigHelper.java [hunk 2]
```diff
@@ -11,32 +11,40 @@
...
 
   private ConfigHelper() {
     throw new RuntimeException("Instantiation of ConfigHelper is not allowed!");
   }
 
   /**
-   * Retrieves projectID from configuration file.
-   * @return projectID; {@code null} if configuration file wasn't found
-   * or it was incorrect/didn't have projectID field
+   * Retrieves sensitive from configuration file.
```
Comment (RazvanBudacaTechquarter) [inline src/main/java/com/google/alpollo/ConfigHelper.java]: Did you mean `Retrieves sensitive data` ?

File: src/main/java/com/google/alpollo/ConfigHelper.java [hunk 3]
```diff
@@ -11,32 +11,40 @@
...
+   * @return String; {@code null} if configuration file wasn't found
+   * or it was incorrect/didn't have this field
```
Comment (RazvanBudacaTechquarter) [inline src/main/java/com/google/alpollo/ConfigHelper.java]: Full stop at the end of this sentence. (Finally I'm not the only one who forgets)

**Review by RazvanBudacaTechquarter — APPROVED** [review]
image

**Review by jgaillard85 — APPROVED** [review]

File: src/main/java/com/google/alpollo/ConfigHelper.java [hunk 4]
```diff
@@ -11,32 +11,42 @@
...
    * used only to get Json object from file.
    */
   private static final class ConfigInfo {
     private String projectID;
+    private String apiKey;
 
     public String getProjectID() {
       return projectID;
     }
+    public String getApiKey() {
```
Comment (jgaillard85) [inline src/main/java/com/google/alpollo/ConfigHelper.java:28]: add empty line between methods

**Review by jgaillard85 — APPROVED** [review]

**Pull request merged**